packages/rancher-istio/1.11/rancher-istio/charts/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: Istio
-  catalog.cattle.io/kube-version: '>= 1.20.0-0 < 1.24.0-0'
```

Comment:
So this should be supported in 1.19, 1.20, 1.21, and 1.22 only?

Reply:
This should support from k8s 1.18 to 1.122 as per the [istio support](https://istio.io/latest/docs/releases/supported-releases/) matrix. Will update to include 1.18